### PR TITLE
Redirect to documentation page for /help and /developer links

### PR DESCRIPTION
**Pull request by travislbrundage**

### CODE REVIEW

File: exchange/views.py [hunk 1]
```diff
@@ -13,10 +13,15 @@
 from geonode.maps.views import _resolve_map
 import requests
 
+
 def HomeScreen(request):
     return render(request, 'site_index.html')
 
 
+def DocumentationPage(request):
```
Comment (sarasafavi) [inline exchange/views.py]: total trivial nitpick, @travislbrundage, but can we snake case this (i.e., `documentation_page`) to be more pythonic/consistent? Feel free to fix L17 above if you like too :)

**Comment by sarasafavi** [comment]
@travislbrundage other than my minor comment above, this LGTM -- if you could fix that whenever you have time, we can get this merged in

**Comment by harts-boundless** [comment]
I rebased and made the suggested change on `https://github.com/harts-boundless/exchange` branch `help-developer`. 
@travislbrundage could you please fetch from that branch, squash the suggested change into your commit to preserve proper attribution, and then push the squashed commit back up to `travislbrundage:help-developer` so it is on this PR and we can merge it?

**Comment by travislbrundage** [comment]
Done.

**Pull request merged**